src/components/utils/dnd-table/dnd-table.js

```diff
@@ -26,7 +26,7 @@ import DndTableBottomLeftButtons from './dnd-table-bottom-left-buttons';
 import DndTableBottomRightButtons from './dnd-table-bottom-right-buttons';
 import { TableNumericalInput } from '../rhf-inputs/table-inputs/table-numerical-input';
 import { TableTextInput } from '../rhf-inputs/table-inputs/table-text-input';
-import { CheckboxInput, SelectInput } from '@gridsuite/commons-ui';
```

Comment:
Im a bit confused: you dont use DndTable anymore in sensi params, so why do you change this component in your PR ?